src/_includes/layouts/contact.liquid

```diff
@@ -1,31 +1,29 @@
 ---
 layout: layouts/base.liquid
 ---
-<div class="bg-parallax-photo">
-  <div
-    :speed-factor="0.5"
-    sectionClass="parallax-outside"
-    containerClass="parallax-inside">
-    <img src="/assets/img/bg-triangle-blue2.png" class="float-right float-md-none" />
+<div class="relative hidden md:block">
+  <div class="absolute top-0 left-0 w-2/5 min-h-[21.875rem] max-w-full overflow-visible z-[-1] ">
```

Comment:
I don't think you need this div on line 5.  Both divs on line 5 and 6 are position absolute.  When an element is position absolute, it's taken out of the flow of the DOM and are positioned to their closest relative parent (line 4).  So, the div on line 5 isn't really doing anything, unless you're setting a background image on it or doing something else with it in a stylesheet.  if that's the case, I wouldn't nest absolute positioned elements unless they have different relative parents.